README.md

```diff
@@ -94,7 +94,9 @@ public class HelloWorld {
         GraphQLSchema schema = GraphQLSchema.newSchema()
                         .query(queryType)
                         .build();
-        Map<String, Object> result = (Map<String, Object>) new GraphQL(schema).execute("{hello}").getData();
+        
+        GraphQL graphQL = GraphQL.newObject(schema).build();
```

Comment:
GraphQL.newGraphQL() I think is more in line with the other builder factory names. (newObject() is already "taken" by GraphQLObjectType)